src/state/contexts/APIContext.js

```diff
@@ -103,6 +105,7 @@ const APIProvider = ({ children }) => {
   //AUTH
 
   const getAuthHeader = authState => {
+    // console.log('getAuthHeader, authState: ', authState);
```

Comment:
make sure to remove the console logs

Reply:
This is done. The comment-out line was removed.